src/ducks/authentication/MobileRouter.jsx

```diff
@@ -24,11 +26,31 @@ import {
 
 export const AUTH_PATH = 'authentication'
 
-export const onLogout = (store, cozyClient, replaceFn) => {
-  const mobile = store.getState().mobile
+const logIfDev = (level, ...logs) => {
+  if (__DEVELOPMENT__) {
+    // eslint-disable-next-line
+    console[level] && console[level](...logs)
+  }
+}
+
+export const onLogout = async (store, cozyClient, replaceFn) => {
+  try {
+    await stopPushNotifications()
+
+    logIfDev('info', 'Stopped push notifications')
```

Comment:
Nit: I think this cannot be stripped from production code. In React code, they directly write `if (__DEV___)` and have code to strip out `warning` function calls.

Reply:
Yes. This can be done later since it is not the subject of the PR.

Reply:
I will do it here since I added these logs in this PR.